packages/bilby.cython/meta.yaml

```diff
@@ -0,0 +1,22 @@
+package:
+  name: bilby.cython
+  version: 0.5.3
+  top-level:
+    - docs
+    - bilby_cython
+source:
+  url: https://files.pythonhosted.org/packages/source/b/bilby_cython/bilby_cython-0.5.3.tar.gz
+  sha256: 44400b2abc6fa592b13d69f33460cb156d6edb48a783f5539ee0a21e1d0a3508
+requirements:
+  host:
+    - numpy
```

Comment:
I guess numpy is not a build dependency. I cannot find it inside the pyproject.toml file of bilby_cython.